.github/workflows/release-tagging.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Tagging
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tagging:
+    runs-on: ubuntu-latest
+    env:
+      EXCLUDED_PACKAGE_OA_RENDERERS: '!./packages/vckit-oa-renderers/'
+      EXCLUDED_PACKAGE_CREDENTIAL_OA: '!./packages/credential-oa/'
+      EXCLUDED_PACKAGE_EXPLORER: '!./packages/demo-explorer/'
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Install pnpm
+      run: npm i pnpm --global
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '20'
```

Comment:
I recommend using the node-version 21
<img width="247" alt="image" src="https://github.com/uncefact/tests-untp/assets/59736798/a67a5581-7ec0-4a6c-a2e2-c1c7504e18ed">